consensus/clique/src/test/java/org/hyperledger/besu/consensus/clique/jsonrpc/methods/DiscardTest.java

```diff
@@ -121,7 +121,7 @@ public void discardWithoutAddress() {
     final Discard discard = new Discard(validatorProvider);
 
     assertThatThrownBy(() -> discard.response(requestWithParams()))
-        .hasMessage("Missing required json rpc parameter at index 0")
+        .hasMessage("Invalid address parameter")
```

Comment:
Wondering if also knowing the parameter index could be useful for users in any of these RPC error? e.g. if you have multiple parameters with values that either are or could look like an Address (i.e. a hash)

Reply:
Hmm, could be useful yeah. I'll update for this PR and we can assess whether we want to apply it across all of the PRs